src/main/java/org/opentripplanner/gtfs/mapping/BookingRuleMapper.java

```diff
@@ -0,0 +1,52 @@
+package org.opentripplanner.gtfs.mapping;
+
+import org.opentripplanner.model.BookingRule;
+import org.opentripplanner.util.MapUtils;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+/** Responsible for mapping GTFS BookingRule into the OTP model. */
+class BookingRuleMapper {
+
+    private Map<org.onebusaway.gtfs.model.BookingRule, BookingRule> mappedBookingRules = new HashMap<>();
+
+    BookingRuleMapper(AgencyAndIdMapper agencyAndIdMapper) {
+        this.agencyAndIdMapper = agencyAndIdMapper;
+        // Some other mappers?
```

Comment:
None needed

Reply:
Changed `BookingRuleMapper` not to call `AgencyAndIdMapper` here.